geojson_to_sqlite/utils.py

```diff
@@ -24,13 +24,18 @@ def import_features(
 ):
     db = sqlite_utils.Database(db_path)
 
+    if pk is None and has_ids(features[:100]):
+        pk = "id"
```

Comment:
Since this is already using `if pk is None` this will already work - I can pass `--pk ''` to the script and this test will fail.